src/ngx_http_lua_bodyfilterby.c

```diff
@@ -87,11 +87,15 @@ ngx_http_lua_body_filter_by_chunk(lua_State *L, ngx_http_request_t *r,
     ngx_chain_t *in)
 {
     ngx_int_t        rc;
+    ngx_chain_t     *cl;
     u_char          *err_msg;
     size_t           len;
 #if (NGX_PCRE)
     ngx_pool_t      *old_pool;
 #endif
+    ngx_http_lua_ctx_t          *ctx;
```

Comment:
Need a blank line before this line when you fail to align the variable name being declared with the previous lines.